webpack/components/Table/EmptyStateMessage.js

```diff
@@ -9,18 +9,19 @@ import PropTypes from 'prop-types';
 import { CubeIcon, ExclamationCircleIcon, SearchIcon } from '@patternfly/react-icons';
 import { global_danger_color_200 as dangerColor } from '@patternfly/react-tokens';
 
-const KatelloEmptyStateIcon = ({ error, search }) => {
+const KatelloEmptyStateIcon = ({ error, search, customIcon }) => {
```

Comment:
const KatelloEmptyStateIcon = ({ error, search, customIcon }) => {
  switch (true) {
    case error: return <EmptyStateIcon icon={ExclamationCircleIcon} color={dangerColor.value} />;
    case search: return <EmptyStateIcon icon={SearchIcon} />;
    case customIcon: return <EmptyStateIcon icon={customIcon} />;
    default:
      return <EmptyStateIcon icon={CubeIcon} />;
  }
};

Reply:
I think something like the above would likely be better for future use.

Reply:
Don't know how I feel about `switch(true)`..I might or might not remember what this does the next time I read it.. :smile: